apps/src/templates/instructions/codeReview/codeReviewDataApi.js

```diff
@@ -19,3 +19,36 @@ export function submitNewCodeReviewComment(commentText, channelId, token) {
     }
   });
 }
+
+export function enablePeerReview(channelId, levelId, scriptId, token) {
+  return $.ajax({
+    url: `/reviewable_projects`,
+    type: 'POST',
+    headers: {'X-CSRF-Token': token},
```

Comment:
why do we need this token?

Reply:
I mirrored this from the other code review API calls - I believe it's needed since authenticating calls didn't work without it and other similar API calls in the Teacher Feedback feature were passing this too. (@bencodeorg had some more context here as well). Though I will re-verify to see what happens if I don't include this token, since some of the authentication logic on the backend has been updated.

Reply:
haha yeah I'll share my digging -- traced back as far as this PR, which says it has something to do with caching?

https://github.com/code-dot-org/code-dot-org/pull/23534

We have these explicit CSRF tags that get embedded on some pages (eg, studio.code.org):
https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/layouts/application.html.haml#L34
https://api.rubyonrails.org/classes/ActionView/Helpers/CsrfHelper.html
But not in Javalab (or other levels). Not sure why?

When I added the first API call to `ReviewTab`, it did not work until I added this token. @sanchitmalhotra126 does the GET work without it?

Reply:
Yep, just tried and without the token I get a 422 error with a response containing this:
```
> ActionController::InvalidAuthenticityToken

app/helpers/read_replica_helper.rb, line 20
-------------------------------------------
...
```